CRM/CiviAwards/Hook/AlterAPIPermissions/Award.php

```diff
@@ -195,4 +218,27 @@ private function getCustomGroupDetails($customGroup) {
     return [];
   }
 
+  /**
+   * Returns the activity type Id.
+   *
+   * @param string $optionValueName
+   *   Option value name.
+   *
+   * @return int
+   *   Activity type Id.
+   */
+  private function getActivityTypeId($optionValueName) {
+    try {
```

Comment:
Same here than in the other class. I think that can work anyway without returning null, but it seems better.